pywbemtools/pywbemlistener/_cmd_listener.py

```diff
@@ -0,0 +1,1176 @@
+    click.option('--protocol', type=click.Choice(['http', 'https']),
+                 metavar='PROTOCOL',
+                 required=False, default=DEFAULT_LISTENER_PROTOCOL,
+                 help=u'The protocol used by the listener (http, https). '
+                 'Default: {}'.format(DEFAULT_LISTENER_PROTOCOL)),
+    click.option('-c', '--certfile', type=str, metavar='FILE',
+                 required=False, default=None,
+                 envvar=_config.PYWBEMLISTENER_CERTFILE_ENVVAR,
+                 help=u'Path name of a PEM file containing the certificate '
+                 'that will be presented as a server certificate during '
+                 'SSL/TLS handshake. Required when using https. '
+                 'The file may in addition contain the private key of the '
+                 'certificate. '
+                 'Default: EnvVar {ev}, or no certificate file.'.
+                 format(ev=_config.PYWBEMLISTENER_CERTFILE_ENVVAR)),
+    click.option('-k', '--keyfile', type=str, metavar='FILE',
+                 required=False, default=None,
+                 envvar=_config.PYWBEMLISTENER_KEYFILE_ENVVAR,
+                 help=u'Path name of a PEM file containing the private key '
+                 'of the server certificate. '
+                 'Required when using https and when the certificate file '
+                 'does not contain the private key. '
+                 'Default: EnvVar {ev}, or no key file.'.
+                 format(ev=_config.PYWBEMLISTENER_KEYFILE_ENVVAR)),
+    click.option('--indi-call', type=str, metavar='MODULE.FUNCTION',
+                 required=False, default=None,
+                 help=u'Call a Python function for each received indication. '
+                 'Invoke with --help-call for details on the function '
+                 'interface. '
+                 'Default: No function is called.'),
+    click.option('--indi-display', is_flag=True,
+                 required=False, default=False,
+                 help=u'Display received indications on stdout. '
+                 'The format can be modified using the --indi-format option. '
+                 'Default: Not displayed.'),
+    click.option('--indi-file', type=str, metavar='FILE',
+                 required=False, default=None,
+                 help=u'Append received indications to a file. '
+                 'The format can be modified using the --indi-format option. '
+                 'Default: Not appended.'),
+    click.option('--indi-format', type=str, metavar='FORMAT',
+                 required=False, default=DEFAULT_INDI_FORMAT,
+                 help=u'Sets the format to be used when displaying received '
+                 'indications. '
+                 'Invoke with --help-format for details on the format '
+                 'specification. '
+                 'Default: "{dif}".'.format(dif=DEFAULT_INDI_FORMAT)),
+    click.option('--help-format', is_flag=True,
```

Comment:
Should we mark these to show that they output help and terminate.  More of a question than request.

Reply:
Added "and exit"